engine/class_modules/paladin/sc_paladin.hpp

```diff
@@ -917,6 +940,11 @@ struct paladin_action_t : public Base
       am *= 1.0 + p() -> buffs.avenging_wrath -> get_damage_mod();
     }
 
+    if ( affected_by.avenging_crusader && p()->buffs.avenging_crusader->up() )
```

Comment:
Is there a reason to not do it like the others? Would seem weird to me that crusade/avenging wrath both use `up()` when avenging crusader uses check

Reply:
![image](https://user-images.githubusercontent.com/25177817/116775870-751fbb80-aa33-11eb-9614-e35cbc402672.png)
photo to directly show what im talking about

Reply:
As a general rule you should use check() in const functions and up()/down() in non-const functions for benefit-tracking (and also only if the benefit tracking affects a single place, otherwise the information you gain from it is somewhat meaningless).

I understand that this might not be practiced in other places as well, it's just that nobody cared to clean it up.

(The more technical explanation for paragraph 1 is that most const action functions can be called an arbitrary number of times through ready() and APL system, so benefit tracking should not be used there)